### ISSUE TITLE
Model Experiments

### ISSUE THREAD

**Issue by FardinHaque60** [FardinHaque60/ParkPredict#2]
## thoughts 
- ~~check experiments in readme~~
- ~~check experiments sarvy put in his branch~~
- ~~consider doing knn, training on entire semester data with neural net~~

## experiment details 
- ~~make datasets to test on, partition into training, test, and validation (choose random weeks for each category)~~
- ~~based on sarvy experiments test guassian regression, neural net, and random forest more thoroughly~~
- experiment on:
  - training on every week of the semester (timestamp normalized)
  - testing on partitioned data
  - training on entire dataset (timestamp not normalized, may have to fill in gaps or only use weeks where all data collected)
- check chatgpt for other models to potentially use 
 
compare model performance by r^2 metric

**Comment by FardinHaque60** [comment]
- Idea: try out training an LLM using the training data as embeddings input. Theory is that LLM has good for auto completion, prediction stuff so if we make large training set with many prediction possibilities it could work well.
   - will have to find way to run substantially sized LLM locally, maybe start one from scratch. Could hack around the architecture to see if we could just use it for the "auto fill" like functionality on our training data
- Sinusoidal curve may not be accurate since fullness ramp up and down are not at same rate.

**Comment by FardinHaque60** [comment]
listed experiments that will be tried in experiments dir in master

**Comment by FardinHaque60** [comment]
completed in dbbee1f2ca3d82d36ceaf51ca3b3464a49de1dd3
